resources/ga-accelerators.go

```diff
@@ -0,0 +1,71 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/globalaccelerator"
+	"github.com/rebuy-de/aws-nuke/pkg/types"
+)
+
+// GAAccelerator model
+type GAAccelerator struct {
+	svc *globalaccelerator.GlobalAccelerator
+	ARN *string
+}
+
+func init() {
+	register("GAAccelerator", ListGAAccelerators)
```

Comment:
Because it is the type/resource Accelerator within the service GlobalAccelerator. The naming is in line with EC2Image, EC2Address and so on. And how should it look in your opinion for GlobalAccelerator with the 3 parts Accelerator, Listeners and EndpointGroups?

Reply:
I find GlobalAcceleratorAccelerator is super ugly. For GlobalAcceleratorListener and GlobalAcceleratorEndpointGroup it is ok.

Reply:
@svenwltr @der-eismann I'm still waiting for a recommendation on how to rename it correctly

Reply:
Sorry for the late response.

I prefer prefixing the resource name with the package name from the AWS SDK. Also I think it is okay to skip the later part for the actual accelerator. Therefore I suggest:

* `GlobalAccelerator`
* `GlobalAcceleratorListener`
* `GlobalAcceleratorEndpointGroup`